Makefile

```diff
@@ -52,6 +53,10 @@ delete-community-operator:
 install-crds:
 	./scripts/deploy-test-crds.sh
 
+install-litmus:
+	oc apply -f https://litmuschaos.github.io/litmus/2.7.0/litmus-2.7.0.yaml
+	oc apply -f ./scripts/operator-ce.yaml
```

Comment:
Move these steps to a ./scripts/deploy-litmus-operator.sh script and call it here, as the rest of this file is doing.